.github/workflows/sdk-integration-close.yml

```diff
@@ -0,0 +1,33 @@
+name: Close Unmerged SDK PRs
+on:
+  pull_request:
+    types:
+    - closed
+jobs:
+  update-sdk-prs:
+    runs-on: ubuntu-latest
+    name: Update SDK PRs
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'public-api') && !github.event.pull_request.merged }}
+    strategy:
+      matrix:
+        repo:
+          - realm-dotnet
+          - realm-js
+          - realm-cocoa
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Find SDK PR
+      id: find-pr-number
+      run: |
+        pull_number=$(curl -s 'https://api.github.com/repos/realm/${{ matrix.repo }}/pulls?head=realm:${{ github.event.pull_request.head.ref }}' -H 'Accept: application/vnd.github.v3+json' | jq '.[0].number')
```

Comment:
Wouldn't something like https://github.com/actions/github-script#using-a-separate-github-token be cleaner?

Reply:
Didn't know about this action, let me try to use that.